README.md

```diff
@@ -1,2 +1,80 @@
-# rfcs
-RFCs for changes to Google Cartographer projects
+# Cartographer RFCs
+[Cartographer RFCs]: #cartographer-rfcs
+
+For changes more substantial than simple bugfixes and documentation improvements
+we ask that these be put through a very lightweight design review process.
+
+"RFC" stands for "request for comments" and are used, e.g.
+[rust-lang](https://github.com/rust-lang/rfcs), to establish a consistent
+and controlled path for new features to enter their project, so that new
+contributors can anticipate the short- and long-term development roadmap of the
+project.
+
+## Table of Contents
+[Table of Contents]: #table-of-contents
+
+  - [Introduction](#cartographer-rfcs)
+  - [Table of Contents]
+  - [When should you write an RFC]
+  - [Motivation for RFCs]
+  - [Before creating an RFC]
+  - [What the process is]
+
+## When should you write an RFC
+[When should you write an RFC]: #when-should-you-write-an-rfc
+
+You should provide an RFC if you intent to make a *significant* change or
+contribution to one of Cartographer's repositories. A change is e.g.
+*significant* if it introduces a new or modifies an existing *concept*. The
+process is really very lightweight so we encourage you to err on the side of
+RFCs.
+
+Make sure to have *one sentence per line* in the RFC as it makes reading diffs
+much easier. The 80 characters per line limit does not apply to RFCs.
```

Comment:
drop second sentence. What 80 chars per line limit?

Reply:
I am used to having the text files I check into Github formatted at 80 chars per line. This is just to clarify that there is no character line limit for RFCs. I updated the sentence accordingly.